src/workers/parser/tests/mapExpression.spec.js

```diff
@@ -178,6 +281,19 @@ describe("mapExpression", () => {
         bindings: false,
         originalExpression: false
       }
+    },
+    {
+      name: "bindings + destructuring without mappings",
+      expression: "({ a } = { a: 4 })",
+      newExpression: "({ a } = {\n a: 4 \n})",
```

Comment:
The following test case fails.
Shall we map the assignment here? I assume we want to have `({ a: _a }) = { a: 4 })`, but `bindings without mapping` does not seem to work as expected, as it does not map anything.

Reply:
Turned out to be caused most likely by `mapOriginalExpression`. Will file a bug report once entirely certain.